File: Lista 1/1/main.go
/*
Faça um programa para calcular o estoque médio de uma peça, sendo que: 
ESTOQUE MÉDIO = (QUANTIDADE_MÍNIMA + QUANTIDADE_MÁXIMA) / 2.
*/

package main

import "fmt"

func main() {
	var min int
	var max int
	fmt.Println("Digite a quantidade minima:")
	fmt.Scan(&min)
	fmt.Println("Digite a quantidade maxima:")
	fmt.Scan(&max)
	resultado := estoqueMedio(min,max)
	fmt.Println("O estoque medio e: ", resultado)
}

func estoqueMedio(min, max int) int{
	estoque := (min + max) / 2
	return estoque
}

File: Lista 1/1/main.c
/*
Faça um programa para calcular o estoque médio de uma peça, sendo que: 
ESTOQUE MÉDIO = (QUANTIDADE_MÍNIMA + QUANTIDADE_MÁXIMA) / 2
*/
#include <stdio.h>
#include <math.h>

int main() {
    int min, max;
    double estoqueMedio;

    printf("Digite a quantidade mínima: ");
    scanf("%d", &min);

    printf("Digite a quantidade máxima: ");
    scanf("%d", &max);  

    estoqueMedio = (min + max) / 2.0;
    if(fmod(estoqueMedio,1.0) == 0.0){
        printf("O estoque médio é: %.0f\n", estoqueMedio);
    }  else {
        printf("O estoque médio é: %.1f\n", estoqueMedio);
    }

    return 0;
}